app/public/css/main.css

```diff
@@ -92,10 +92,10 @@ html, body {
 /*
 4) NUGGET-INFO
 */
-.nugget-info-tab {
-  display: none;
+.nugget-info-tab.visible {
```

Comment:
aren't we having the animation where the tab scrolls up when it becomes visible?